build/klh10/config.203

```diff
@@ -967,6 +967,11 @@ DEFOPT XBL==30.		; # TCP network channels
 DEFOPT IMPP==1		; Has an IMP interface
 DEFOPT SSIMP==1		; Hooked to "Simulated Simple IMP"
 
+DEFOPT CHAOSP==1	;Has CHAOS net
+DEFOPT MYCHAD==177002	;CHAOS net address
```

Comment:
The chaosnet address of the host should not be hardcoded, but should come from build-time options.

Reply:
The main goal for this pull request is to make it easy for people to get started with a Chaosnet.  It's still possible for experts to make your own ITS with another network configuration.